tasks/5.test.js

```diff
@@ -0,0 +1,12 @@
+const { wordSearch } = require("./5");
+
+describe("Task #5", () => {
+  describe("wordSearch()", () => {
```

Comment:
Wrapping the test suite in an additional describe for `task` is sort of redundant as you have no other functions to test. 🤔